http-server.md

```diff
@@ -491,12 +491,12 @@ func main() {
 ```
 
 If you run `go build` again and hit the same URL you should get `"123"`. Not great, but until we store data that's the best we can do.
+It also didn't feel great that our main application was starting up but not actually working. We had to manually test to see the problem.
 
 We have a few options as to what to do next
 
 -   Handle the scenario where the player doesn't exist
 -   Handle the `POST /players/{name}` scenario
--   It didn't feel great that our main application was starting up but not actually working. We had to manually test to see the problem.
```

Comment:
This isn't really an option, more a description of a current drawback. Moved a few lines above